src/app/shared/human-labels.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'humanLabels'
+})
+export class HumanLabelsPipe implements PipeTransform {
```

Comment:
What do you think to make it more general - not only for labels.
Fo example HumanizeSymbolPipe ?